hw1/README.rst

```diff
@@ -0,0 +1,56 @@
+======================
```

Comment:
Any changes outside `hw0` branch should not be included in a PR against that branch.